src/stories/containers/TransparencyReport/transparencyReportUtils.tsx

```diff
@@ -107,14 +113,18 @@ export const RenderNumberWithIcon = (data: TargetBalanceTooltipInformation) => {
         {!isMobileResolution && (
           <Container>
             {showIconToolTip && (
-              <CustomerPopoverExtends
-                widthArrow
+              <ExtendedCustomPopover
+                handleShowPopoverWhenNotSpace={handleShowPopoverWhenNotSpace}
+                refElementShowPopover={hrefPopoverElement}
                 sxProps={{
                   '.MuiPaper-root.MuiPaper-elevation.MuiPaper-rounded': {
                     overflowX: 'unset',
                     overflowY: 'unset',
+                    marginTop: marginTopPopoverPosition ? 3 : -3,
                   },
                 }}
+                widthArrow
+                // hasSpacePositionArrow={marginTopPopoverPosition}
```

Comment:
is this comment required here?

Reply:
My bad, its part of the code, let me uncomment